plugins/contactpressuresensorsarray/src/ContactPressureSensorsArrayDriver.cpp

```diff
@@ -0,0 +1,516 @@
+/*
+ * Copyright (C) 2018 Fondazione Istituto Italiano di Tecnologia RBCS & iCub Facility & ADVR
+ * Authors: see AUTHORS file.
+ * CopyPolicy: Released under the terms of the LGPLv2.1 or any later version, see LGPL.TXT or LGPL3.TXT
+ */
+
+#include "ContactPressureSensorsArrayDriver.h"
+#include <GazeboYarpPlugins/Handler.hh>
+#include <GazeboYarpPlugins/common.h>
+
+#include <yarp/os/LockGuard.h>
+#include <yarp/os/Log.h>
+#include <yarp/os/LogStream.h>
+#include <yarp/math/Math.h>
+#include <yarp/math/SVD.h>
+
+#include <sdf/Param.hh>
+
+#define DEBUG 0
+#define N_AXIS 3
```

Comment:
This is used only in `GazeboYarpContactPressureSensorsArrayDriver::prepareMappingMatrix()`, so I would transform it in a local const variable of that method. In general it is better to avoid to use the preprocessor unless it is strictly necessary.